package.json

```diff
@@ -18,6 +18,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@oada/types": "^1.0.6",
+    "@overleaf/o-error": "^3.2.0",
```

Comment:
Looks like `@overleaf/o-error` is currently not being used (although it is loaded by `http.ts`). If that's the case, I would remove the dependency on this package.